airbyte-webapp/src/hooks/services/useBuildUpdateCheck.ts

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from "react";
+import { useIntl } from "react-intl";
+import { fromEvent, interval, merge, throttleTime } from "rxjs";
+
+import { useNotificationService } from "./Notification";
+
+interface BuildInfo {
+  build: string;
+}
+
+const INTERVAL = 60 * 1000;
```

Comment:
I'm a fan of this syntax, but take it or leave it 😃 

```suggestion
const INTERVAL = 60_000;
```